src/components/Description.jsx

```diff
@@ -1,24 +1,33 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import {
-    Link,
-    useParams,
+  Link,
+  useParams,
 } from 'react-router-dom';
+import Axios from 'axios';
 import BorderTopCard from './BorderTopCard';
 
-
 const Description = () => {
-    const { descriptionId } = useParams();
-    
-    return (
-        <div>
-            <Link to="/boxes/1">
-                <BorderTopCard />
-            </Link>
-            <h1>
-                hello from { descriptionId }
-            </h1>
-        </div>
-    )
-}
+  const { descriptionId } = useParams();
+  const [description, setDescription] = useState({});
+
+  useEffect(() => {
+    Axios.get(`http://localhost:8000/boxes/${descriptionId}`)
+      .then((response) => {
+        setDescription(response.data);
+      });
+  }, [descriptionId]);
+
+  return (
+    <div>
+      <Link to="/boxes/1">
+        <BorderTopCard />
+      </Link>
+      <h1>
+        hello from
+      </h1>
+      <p>{description !== {} ? description.name : 'no content'}</p>
```

Comment:
`description !== {}` is always true, because each Object is a different value in JS

don't give a defined default value to your state:

    const [description, setDescription] = useState();

so you can write:

    description != null ? description.name : 'no content'